src/zc/relation/catalog.py

```diff
@@ -1177,13 +1180,13 @@ def canFind(self, query, maxDepth=None, filter=None,
         if getQueries is None:
             queryFactory, getQueries = self._getQueryFactory(
                 query, queryFactory)
-        try:
-            next(self.yieldRelationTokenChains(
+
+        _ = next(self.yieldRelationTokenChains(
                 *self._parse(
                     query, maxDepth, filter, targetQuery,
                     targetFilter, getQueries) +
-                (False,)))
-        except StopIteration:
+                (False,)), _marker)
```

Comment:
@ale-rt 
This line and lines 1185-1187 above need to be dedented by 4 spaces to the left.
Once we replaced `try-except` with the assignment this whole expression needs one indentation level less.

Reply:
@ale-rt Of course. It's not necessary, but it's changing the code style of the package. A small nit. 🙂 
I just wanted to bring it to your attention.
I see that you're preparing other changes, so this small style fix could perhaps be included in the upcoming changes.
I'll resolve the conversation.
Thanks!